### ISSUE TITLE
WER for the loanword when validating code-switching ASR model

### ISSUE THREAD

**Issue by kthworks** [NVIDIA/NeMo#6650]
Hello, I have question for the Error rate for the code-switching ASR model.

There are many loanwords in korean which from english.
For example, "Coffee" is "커피" in korean. These two words have same pronunciation and meaning.

The sentence "커피 마실래?" is "would you want coffee?" in english.

Let's say there is some audio input, and there are two situation in output.

1. "커피 마실래?"
2. "coffee 마실래?"

Actually, both two sentence are correct and natural for korean people. 

Then, how to calculate error rate?
if the label is '커피' then 'coffee' is wrong?
or, '커피' and 'coffee' are both correct?

**Comment by titu1994** [comment]
Wer alone is not a sufficient metric in this case. It makes no distinction between equivalent sentences. You might try something like sacrebleu as a secondary metric but I do not know how valid it would be.

**Comment by kthworks** [comment]
I have additional questions.

Then, in pre-trained Multilingual model such as "stt_enes_conformer_ctc_large", how did you calculate WER? 
( https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_enes_conformer_ctc_large )

it was calculated by just matched each words to "label" words?

Always thank for your immediate and kindness reply.

**Comment by titu1994** [comment]
Yes, that is a multi lingual model but not a code switched model. Code switched models are evaluated against just wer - so if the above case occurs we count it as a mistake.

**Comment by kthworks** [comment]
Thank you for reply :)
